torlak_accent.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+import argparse
+import datetime
+import os
+import re
+os.environ.setdefault("TF_CPP_MIN_LOG_LEVEL", "2")  # Report only TF errors by default
+
+from tqdm.auto import tqdm
+import numpy as np
+import tensorflow as tf
+import time
+from torlakian_data import TorlakianData
+
+os.environ['CUDA_VISIBLE_DEVICES'] = '-1'
+
+# Based on: https://github.com/ufal/npfl114/blob/master/labs/03/uppercase.py
+
+parser = argparse.ArgumentParser()
```

Comment:
be consistent. In the file below, you have argparser as part of __main__.

Reply:
Not sure which file you are referring to. In the file `model.py` the situation is the same: ArgumentParse is instantiated outside of main, and used as an argument to the `main` method. Right now, `model.py` and `torlak_accent.py` are the same in that regard.

Reply:
Yes but look at `torlakian_data.py` There it is under __main__

Reply:
Fixed